package.json

```diff
@@ -43,7 +43,7 @@
     "getusermedia": "^1.3.7",
     "git-commits": "^1.2.0",
     "git-parse-commit": "^1.0.0",
-    "gl": "4.0.1",
+    "gl": "4.9.2",
```

Comment:
Wondering why this change is necessary?
If not, please revert it.

Reply:
Yes this was in the process of debugging the build, it is not necessary change. Reverting.